gradle.properties

```
@@ -1,5 +1,5 @@
 group=org.sonarsource.scanner.gradle
-version=4.2.1-SNAPSHOT
+version=4.3.0-SNAPSHOT
```

Just 4.3 is enough, omit the trailing 0
```suggestion
version=4.3-SNAPSHOT
```